src/lib/StorageFileApi.ts

```diff
@@ -178,6 +178,29 @@ export class StorageFileApi {
     }
   }
 
+  /**
+   * Returns a publicly accessible URL
```

Comment:
Change this to `Retrieve URLs for assets in public buckets`? That should make it obvious that this is only valid for objects in public buckets.

Reply:
Thanks @inian ! On it right now!